OrbitLinuxTracing/TracerThread.h

```diff
@@ -178,6 +180,8 @@ class TracerThread {
     uint64_t gpu_events_count = 0;
     uint64_t lost_count = 0;
     absl::flat_hash_map<PerfEventRingBuffer*, uint64_t> lost_count_per_buffer{};
+    std::shared_ptr<std::atomic<uint64_t>> discarded_out_of_order_count =
+        std::make_unique<std::atomic<uint64_t>>(0);
```

Comment:
`shared_ptr... = make_unique`... is this intended?

Reply:
Nope, it was wrong in the lines below and I copy-pasted the mistake, go me. Anyway, switched away from the `shared_ptr`s altogether.

Reply:
Thanks!